### PR TITLE
no unshallow

### PR DESCRIPTION
**Pull request by pat-s**
due to https://ci.woodpecker-ci.org/repos/8958/pipeline/183/12

### CODE REVIEW

**Comment by pat-s** [comment]
waiting for #57

**Review by anbraten — COMMENTED** [review]
I think this is needed for the release-helper to get previous commits to determine the new version

**Comment by pat-s** [comment]
But it appears in the `test-chart` step which is unrelated to any release-helper action?

**Comment by anbraten** [comment]
Ahh but I think we needed it for `ct` as well as its checking which chart was edited.

File: .woodpecker/test.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ steps:
   test-chart:
     image: quay.io/helmpack/chart-testing
     commands:
-      - git fetch origin main --unshallow --no-tags
+      - git fetch origin main --no-tags
```
Comment (anbraten) [inline .woodpecker/test.yml:16]:
```suggestion
      - git fetch origin main --unshallow --no-tags || true
```

Comment (pat-s) [reply .woodpecker/test.yml:16]: If we do so, we should also add a comment why this was done :)

Comment (anbraten) [reply .woodpecker/test.yml:16]: sounds like a good idea

**Review by anbraten — APPROVED** [review]

**Pull request merged**